### DISCUSSION TITLE
Languages arent being created properly in repository

### DISCUSSION THREAD

**Discussion by v0ncent** [lowlighter/metrics#1228] [Q&A]
Hello!

I just recently set up metrics for my current working project, but I believe I have set up it incorrectly and I am not sure on how to properly fix it. 

I am trying to set up Most Used Languages for my repo but it isn't registering them properly.
![metrics plugin languages](https://user-images.githubusercontent.com/83373495/189756403-1b057120-569f-4903-b3fe-204eae9617a3.svg)

As you can see above is what the work flow is outputing.
My workflow:
```name: Metrics-Languages
on:
  # Schedule daily updates
  schedule: [{cron: "0 0 * * *"}]
  # (optional) Run workflow manually
  workflow_dispatch:
  # (optional) Run workflow when pushing on master/main
  push: {branches: ["master", "main"]}
jobs:
  github-metrics:
    runs-on: ubuntu-latest
    permissions:
      contents: write
    steps:
      - uses: lowlighter/metrics@latest
        with:
          filename: metrics.plugin.languages.svg
          token: ${{ secrets.METRICS_TOKEN }}
          template: repository
          repo: v0ncentArtSite
          plugin_languages: yes
          plugin_languages_indepth: yes
```
I am not sure on how to fix the above issue, so any help would be greatly apreciated.

**Comment by lowlighter** [comment]
Hello 👋 !

It should work with the workflow you provided indeed, I'll try to reproduce it later in the next days to see what's causing  👍

**Reply by v0ncent** [reply]
Appreciate it ! If you need more info just ask!

**Reply by lowlighter** [reply]
Sorry for the lack of follow-up
There's currently a refactor of the indepth languages option (#1118) that should be merged in the next few days, hopefully it'll solve your issue 🙂 !

**Reply by v0ncent** [reply]
Alright sounds good! Thanks for the help and follow up!
